day_6/exercises/burrito.rb

```diff
@@ -5,15 +5,33 @@
 # 3. change_protein
 
 class Burrito
-  attr_reader :protein, :base, :toppings
+  attr_accessor :protein, :base, :toppings
   def initialize(protein, base, toppings)
     @protein  = protein
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping(toppings)
+    @toppings = toppings
```

Comment:
This sets the instance variable to whatever you pass in, but we'd rather see you use an array method that adds the parameter that's passed in to the array. Can you find an Array method that can do that?

Reply:
Honestly Tim I'm not sure what you mean by "an array method that adds the parameter that's passed into the array." or how to do that. It wasn't specified in the exercise that this was the expectation. 
I'm currently on the road/camping and won't be back until late Sunday evening. I don't think I can accommodate this change until I better understand what it is you're asking or how to do it.